modules/django-files/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Files (Django)",
+  "description": "Backend for the Files module.",
+  "root": "/backend/modules"
```

Comment:
@danielsousaio There are some modules that point to the subdir, i.e. https://github.com/crowdbotics/modules/blob/master/modules/django-push-notifications/meta.json. This one is taken from django-articles, wouldn't be "/backend/modules/django-files" the right one? If so, django-articles should also be fixed

Reply:
This should be `/backend/modules` so that upon installation you'll get `files` and `migrations` copied to that path:
```
~$ ls /backend/modules
files migrations
```

Reply:
That push-notifications example is different, notice how the app files are on the root of the module.